### PR TITLE
added lesson of Grove

### PR DESCRIPTION
**Pull request by hansonCc**
added 1,2 lesson of Grove

### CODE REVIEW

File: PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Description:
```
Comment (jadonk) [inline PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md]: I'd like this to be simply part of Kit.md. The instructions are pretty short and I don't see any reason we can't have them all in one file. Eliminating the directories is especially useful for organizing Python modules.

File: PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+In this lesson, students will light up the RGB LED, and learn how to use Slide Potentiometer and Rotary Angle Sensor to change the light of RGB LED.
+
+## Hardware Requirement:
```
Comment (jadonk) [inline PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md]: Do we need a separate requirement list given that the setup is simply connecting the required devices? Do we need instructions on printing out the tutorial at the very beginning and encouraging users to shut the power down before making connections?

File: PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- Plug the Grove - Rotary Angle Sensor into **A5** port
+- Grove - Chainable RGB LED into **A2** port
+- Power the Pocket Beagle via the **micro USB** port
+
+![](../img/_DAS6312-1.jpg)
+
+
+## Software
+
+- Step 1. Enter Cloud9 by typing IP of Pocket Beagle
```
Comment (jadonk) [inline PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md]: PocketBeagle

We'll need to have general instructions up-front on finding PocketBeagle's IP address. I'd like a little script that prints it out on the LCD, perhaps configuring it to show on startup. https://www.geeksforgeeks.org/display-hostname-ip-address-python/

File: PocketBeagle/Grove/Lesson-2.Musical-Note/Musical_Note.py [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+import os
+import sys
+import wave
+import pyaudio
+from tqdm import tqdm
+try:
+    import iio
+except:
+    # By default the iio python bindings are not in path
+    sys.path.append('/usr/lib/python2.7/site-packages/')
```
Comment (jadonk) [inline PocketBeagle/Grove/Lesson-2.Musical-Note/Musical_Note.py]: nope.

File: PocketBeagle/Grove/Lesson-2.Musical-Note/Musical_Note.py [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+   'fa.wav',
+   'so.wav',
+   'la.wav',
+   'ti.wav'
+   ]
+def Play_Music(file):
+    # define stream chunk
+    chunk = 1024
+    # open a wav format music
+    f = wave.open(file,"rb")
```
Comment (jadonk) [inline PocketBeagle/Grove/Lesson-2.Musical-Note/Musical_Note.py]: Can we move these files to /tmp ?

File: PocketBeagle/Grove/Lesson-2.Musical-Note/tone_generator.py [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+            time.sleep(0.1)
+    tone_freq = [ v for v in sorted(tone_freq_map.values())]
+    for i in range(len(tone_freq)):
+        f = wave.open( "/home/debian/scale/%s"%_SCALE_DEFS[i],"wb")
+        f.setnchannels(channels)
+        f.setsampwidth(sampwidth)
+        f.setframerate(framerate)
+        f.writeframes(synthesizer(tone_freq[i]).tostring())
+        f.close()
+        print("/home/debian/scale/%s generated successfully"%_SCALE_DEFS[i])
```
Comment (jadonk) [inline PocketBeagle/Grove/Lesson-2.Musical-Note/tone_generator.py]: Can we move these to /tmp ?

File: PocketBeagle/Grove/Lesson-3.Switch-the-Music/Switch_the_Music.py [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+except:
+    # By default the iio python bindings are not in path
+    sys.path.append('/usr/lib/python2.7/site-packages/')
+    import iio
+def GetCmdReturn(cmd):
+    r = os.popen(cmd)
+    text = r.read() 
+    r.close()
+    return text.strip('\n')    
+class JHD1802:
```
Comment (jadonk) [inline PocketBeagle/Grove/Lesson-3.Switch-the-Music/Switch_the_Music.py]: Again, use a module (LCD.py).

File: PocketBeagle/Grove/Lesson-4.Download-Music-via-the-WIFI-dongle/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+if you don't know how to use `connmanctl`, maybe you should review the previous lessons
+
+- Step 2.Download [winscp](https://winscp.net/eng/download.php).
+
+- Step 3. Open winscp and type the hostname and username 
+
+![](../img/winscp.png)
+the hostname is an IP address of Pockbeagle that can use ifconfig to find it. The username is `debian`
+
+- Step 4. Drag your music file to `/home/debian/scale`(Please select *.wav to /home/debian/scale)
```
Comment (jadonk) [inline PocketBeagle/Grove/Lesson-4.Download-Music-via-the-WIFI-dongle/README.md]: Looks like the * might be interpreted as an emphasis.

**Comment by jadonk** [comment]
I'm merging here and will move into .work-in-progress and apply other fixes.

File: PocketBeagle/Grove/Lesson-2.Musical-Note/Musical_Note.py (hunk 5 above)
Comment (hansonCc) [reply PocketBeagle/Grove/Lesson-2.Musical-Note/Musical_Note.py]: if we move in /tmp , the file will disappear when we restart it.

File: PocketBeagle/Grove/Lesson-4.Download-Music-via-the-WIFI-dongle/README.md (hunk 8 above)
Comment (hansonCc) [reply PocketBeagle/Grove/Lesson-4.Download-Music-via-the-WIFI-dongle/README.md]: Yes, I will modify  * to xxx.

**Pull request merged**